study.md

```diff
@@ -66,15 +67,15 @@ nearest_star = the_sun
 Will modifying `sol` affect `the_sun`? Why or why not?
 
 ```md
-<!-- your answer here -->
+no, its a different instance of the class Star
 ```
 
 ## Ruby Object Equality: Part II
 
 Will modifying `the_sun` affect `nearest_star`? Why or why not?
 
 ```md
-<!-- your answer here -->
+no
```

Comment:
Modifying `the_sun` will affect `nearest_star` and vice versa because both are referencing the same object in memory.